tools/knarc/Makefile

```diff
@@ -8,7 +8,7 @@ else
 C_SRCS   :=
 UNAME_S  := $(shell uname -s)
 ifeq ($(UNAME_S),Darwin)
-LDFLAGS  += -lstdc++ -lc++ -lc
+LDFLAGS  += -lstdc++ -lc++ -lc /usr/local/opt/llvm@8/lib/libc++fs.a
```

Comment:
this is a really really big assumption, how do you know that llvm is even installed on the system etc etc? especially in that path

Reply:
Inconsequential, since you can't build on Mac anyway :morphon: